compiler/nim.nim

```diff
@@ -95,9 +95,15 @@ proc handleCmdLine(cache: IdentCache; conf: ConfigRef) =
       var cmdPrefix = ""
       case conf.backend
       of backendC, backendCpp, backendObjc: discard
-      of backendJs: cmdPrefix = findNodeJs() & " "
+      of backendJs:
+        # D20210217T215950:here this flag is needed for node < v15.0.0, otherwise
+        # tasyncjs_fail` would fail, refs https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode
+        cmdPrefix = findNodeJs() & " --unhandled-rejections=strict "
```

Comment:
Can't we require node >= 15.0 instead?

Reply:
15.0 was release in 2020-10-20, see https://nodejs.org/en/download/releases/

also CI for azure, github actions use 12.x (only openbsd or freebsd (or both, I forgot) use >= 15.x

Reply:
=> https://github.com/nim-lang/Nim/pull/18003